### DISCUSSION TITLE
Wrong Changelog formatting from summaries

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [changesets/changesets#1269] [Q&A] [answered]
when I run `npx changeset` and in the summary question I press enter to open an editor to write longer summary.

In the text editor I write this 

I tried this:
```
- Something something 
- Something something 
- Something something 
- Something something 
```
and got this:
```
 -- Something something 
  - Something something 
  - Something something 
  - Something something 
```

**Comment by zaaakher** [comment, marked as answer]
You can make your own formatter that resolve your issue.

Following this [page](https://github.com/changesets/changesets/blob/main/docs/modifying-changelog-format.md) of the docs you can create the file `.changeset/custom-formatting.js` and in your `.changeset/config.json` file you add the following line:
```
  "changelog": "./customer-formatting.js",
```

and your custom formatter can be whatever you like it to be. And if you want a clear no-format here's a [gist](https://gist.github.com/zaaakher/9ee450156bc71dd8948b7a1a143e7d18#file-changeset-formatting-js) for it

**Reply by TheMikeyRoss** [reply]
Oh man thank you so much for the gist 👍
